cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	//"runtime/pprof"
+
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/lpms/ffmpeg"
+	"github.com/livepeer/m3u8"
+)
+
+func main() {
+	/*
+		cprof, err := os.Create("bench.prof")
+		if err != nil {
+			panic(err)
+		}
+		pprof.StartCPUProfile(cprof)
+		defer pprof.StopCPUProfile()
+	*/
+	// Override the default flag set since there are dependencies that
+	// incorrectly add their own flags (specifically, due to the 'testing'
+	// package being linked)
+	flag.Set("logtostderr", "true")
+	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+
+	fname := flag.String("fname", "", "Input m3u8 manifest file")
+	conc := flag.Int("conc", 1, "# of concurrent transcode sessions")
+	segs := flag.Int("segs", 0, "Maximum # of segments to transcode (default all)")
+	profs := flag.String("profs", "P240p30fps16x9,P360p30fps16x9,P720p30fps16x9", "Transcoding options for broadcast job, or path to json config")
```

Comment:
`transcodingOptions` to match the naming convention used in [livepeer](https://github.com/livepeer/go-livepeer/blob/7f9534c365123f455c7fd3bfb9ebcbf5a99cdd43/cmd/livepeer/livepeer.go#L124)?